docs/Superoperator representations.md

```diff
@@ -12,49 +16,565 @@ where $i$ is a row and $j$ is a column index.
 
 We define `vec` to be column stacking
 
-$ {\rm vec}(A) = (a_{11},a_{21},\ldots,a_{m1},a_{12},\ldots,a_{mm})^T$
+$$ |A\rangle \rangle :={\rm vec}(A) = (a_{11},a_{21},\ldots,a_{m1},a_{12},\ldots,a_{mm})^T \quad (1) $$
 
 were $T$ denotes a transpose. Clearly an inverse operation, `unvec` can be defined so that 
 
-$ {\rm unvec}\big ( {\rm vec}(A) \big ) = A$.
+$$ {\rm unvec}\big ( {\rm vec}(A) \big ) = A.$$
```

Comment:
`unvec()` generally depends on the dimensions of A, which are not recoverable from `vec(A)`.  I know we will often focus on square `A`, but for generality, we should require the dimensions for `A`, defaulting to the sqrt of the dimension of `vec(A)` (when possible).

Reply:
Good idea. As it will require changes to the code i'll make a check point. But I'll change the doc now.

- [ ] todo change code

Reply:
done by kyle